### PR TITLE
feat(component): Notification

### PR DESCRIPTION
**Pull request by 2creative**
Issue: https://github.com/action-is-hope/shelley/issues/121

### CODE REVIEW

File: src/Notification/index.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+export * from "./Notification";
```
Comment (LucjaWszola) [inline src/Notification/index.ts:1]: just missing an empty line

File: src/styles/default/notification.st.css [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    height: 100%;
+    background-color: grey;
+  }
+
+  Notification::closeButton {
+    position: absolute;
+    right: var(--spacing-unit);
+  }
+
+}
```
Comment (LucjaWszola) [inline src/styles/default/notification.st.css:52]: empty line

File: src/styles/shelley/notification.st.css [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+ * CSS API: https://github.com/action-is-hope/shelley/blob/main/src/Notification/notification.st.css
+ * 
+ */
+@st-import Shelley from "./project.st.css";
+@st-import Notification from "../../Notification/notification.st.css";
+
+@st-scope Shelley {
+  Notification {}
+
+}
```
Comment (LucjaWszola) [inline src/styles/shelley/notification.st.css:15]: empty line

**Comment by LucjaWszola** [comment]
Hey @2creative, it looks good, few empty lines missing.
Also, RTL does not seem to be working... possible to fix it?

**Review by LucjaWszola — APPROVED** [review]

**Comment by andreadsw** [comment]
Hi @2creative, It looks good. The testing code looks amazing. There is a lot of effort. I have only one question before I approve. Is there a way to improve the icon's font size? It looks small at mom. What do you think?   

Notification-Icon


Notification-Icon-2em

**Comment by 2creative** [comment]
> Hi @2creative, It looks good. The testing code looks amazing. There is a lot of effort. I have only one question before I approve. Is there a way to improve the icon's font size? It looks small at mom. What do you think?
> 
> Notification-Icon
> 
> Notification-Icon-2em

Thanks @andreadsw I just made the icon size same as the toast component size to be consistent.

**Review by andreadsw — APPROVED** [review]
Cool. 🎉 Thank you for clarifying. It looks good for me.

**Review by GaxZE — CHANGES_REQUESTED** [review]
@2creative can we add the documentation for the different variations. Currently every single one is:

```
<Notification />
``

**Comment by 2creative** [comment]
> documentation

Which file are you referring to?

**Comment by GaxZE** [comment]
@2creative hit code on that screenshot.

**Comment by 2creative** [comment]
> @2creative hit `show code` on that screenshot.
> 
> See Button for example: Screenshot 2023-07-26 at 14 47 52 Screenshot 2023-07-26 at 14 47 45

Thanks for looking. Updated the source examples.

**Pull request merged**